app/liquid_tags/user_subscription_tag.rb

```diff
@@ -1,9 +1,242 @@
 class UserSubscriptionTag < LiquidTagBase
+  PARTIAL = "liquids/user_subscription".freeze
+
+  SCRIPT = <<~JAVASCRIPT.freeze
```

Comment:
In case you missed it, check out my PR description where I explain why this is set up this way 🙃 .